misaligned.c

```diff
@@ -1,13 +1,24 @@
 #include <stdio.h>
 #include <assert.h>
 
-int printColorMap() {
-    const char* majorColor[] = {"White", "Red", "Black", "Yellow", "Violet"};
-    const char* minorColor[] = {"Blue", "Orange", "Green", "Brown", "Slate"};
+const char* majorColor[] = {"White", "Red", "Black", "Yellow", "Violet"};
+const char* minorColor[] = {"Blue", "Orange", "Green", "Brown", "Slate"};
+
+void testColorPair(int MjIndex, int MnIndex, const char * MjColor, const char * MnColor)
+{
+    int checkIndex = 0;
+    assert(majorColor[MjIndex] == MjColor);
+    assert(minorColor[MnIndex] == MnColor);
+    checkIndex = (MjIndex * MnIndex) / 10;
+    assert(checkIndex == 1);
+}
+
+int printColorMap() {  
     int i = 0, j = 0;
     for(i = 0; i < 5; i++) {
         for(j = 0; j < 5; j++) {
             printf("%d | %s | %s\n", i * 5 + j, majorColor[i], minorColor[i]);
+            testColorPair(i,j,majorColor[i],minorColor[i]);

```

Comment:
test code is part of production code. the same code gets shipped to field as deployment. This can cause unwanted behavior from this software.

Please separate test code and production code. Production code should not have any dependency on test code.
Whereas test code should consume production code i.e. provide input and extract output and verify.